### PR TITLE
Update code style and raise violation to warning

### PR DESCRIPTION
**Pull request by bgavrilMS**

### CODE REVIEW

**Review by jmprieur — COMMENTED** [review]
Did you fix the warnings associated with this raise?

**Comment by bgavrilMS** [comment]
@jmprieur - I did not. I think the warnings are not fired in CI builds, but only on local builds.

**Review by jennyf19 — COMMENTED** [review]
+1 to @jmprieur 's comment on fixing the warnings.

**Comment by bgavrilMS** [comment]
@jennyf19 @jmprieur - that sounds fair, will do it tomorrow.

**Review by neha-bhargava — APPROVED** [review]
Works for me :)
Thanks!

File: .editorconfig [hunk 1]
```diff
@@ -30,84 +31,104 @@ csharp_new_line_between_query_expression_clauses = true
...
-csharp_style_var_for_built_in_types = false:none
+# Types: use keywords instead of BCL types, and permit var only when the type is clear
+csharp_style_var_for_built_in_types = false:warning
 csharp_style_var_when_type_is_apparent = false:none
-csharp_style_var_elsewhere = false:suggestion
-
-# use language keywords instead of BCL types
-dotnet_style_predefined_type_for_locals_parameters_members = true:suggestion
-dotnet_style_predefined_type_for_member_access = true:suggestion
+csharp_style_var_elsewhere = false:warning
```
Comment (jennyf19) [inline .editorconfig:50]:
```suggestion
csharp_style_var_elsewhere = false:warning
csharp_style_var_for_built_in_types = false:none
```

Comment (jennyf19) [reply .editorconfig:50]: idk if this will help w/the warnings. i can't get the editor config to work locally and pick up this change.

**Comment by henrik-me** [comment]
+1 warnings should be fixed and these should run in CI/release builds ensuring no violations hits the repo.

**Pull request closed without merging**